extensions/dicom-pdf/src/tools/modules/dicomVideoModule.js

```diff
@@ -0,0 +1,61 @@
+import cornerstone from 'cornerstone-core';
```

Comment:
This file is called dicomVideoModule. What does it do? Is it different than the dicom-video extension? why we are calling it module? (since we have extension's modules and that might be not ideal naming)

Reply:
Deleted it